specsim/config.py

```diff
@@ -370,7 +374,7 @@ def load_table(self, parent, column_names, interpolate=True, as_dict=False):
             # Look for parent.table.path first.
             paths.append(os.path.join(self.abs_base_path, node.path))
         except AttributeError:
-            path_keys = node.paths.keys()
+            path_keys = list(node.paths.keys())
```

Comment:
Sorry - I edited my comment above.  My confusion was because the github email only shows the lines *above* your comment, not below.  I guess that means you are supposed to insert comments below the relevant code?